.build/before_build.ps1

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/pwsh
+
+If ($isWindows) {
```

Comment:
Any particular reason to have separate `install` and `before_build` scripts, rather than merge into one?

Reply:
AppVeyor starts the services after `install` but before `before_build`, so this gets around starting/restarting PostgreSQL.

Reply:
Hmm, OK... If the only thing we need to do before the build is create the user, then why not just tell the tests to use the default `postgres` user via the `NPGSQL_TEST_DB` environment variable? This is how it currently works, no need to create a user or anything...